types/react-redux-epic/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-unnecessary-generics": false
```

Comment:
I think you can move the rule to code as a comment like `tslint:disable-next-line...`

Reply:
I updated the code and added tests